test/new-e2e/tests/containers/k8s_test.go

```diff
@@ -259,7 +259,7 @@ func (suite *k8sSuite) TestClusterAgentConfigCheck() {
 		})
 		suite.Require().NoError(err)
 		suite.Require().Len(linuxPods.Items, 1)
-		stdout, stderr, err := suite.podExec("datadog", linuxPods.Items[0].Name, "cluster-agent", []string{"agent", "configcheck", "-n"})
+		stdout, stderr, err := suite.podExec("datadog", linuxPods.Items[0].Name, "cluster-agent", []string{"agent", "configcheck"})
```

Comment:
`-n` was for "no color". 
does it means that now by default the command return with "no color" ? 
should we update the test validation to verify that now we have some "color" in the response

Reply:
The test original was without the `-n` flag. I added flag here https://github.com/DataDog/datadog-agent/pull/26435/commits/e9b8b9d4ae3438f2f8cab1e8b248d0815f07b9ca. While I fixed CI for the color output I broke the color output for TTY, which by default is disable by the color package https://github.com/fatih/color/blob/main/color.go#L22-L23

The test run in TTY mode so by default the test are without color. That is why I removed the flag to ensure the TTY behaviour is correct

> does it means that now by default the command return with "no color" ?
By default the command returns color.

> should we update the test validation to verify that now we have some "color" in the response
We could, the original test did not included that color version. Do you want it as part of this PR?